sensor/src/main.py

```diff
@@ -27,14 +27,15 @@ def run() -> None:
     - Check for configuration update
     """
 
-    logger = utils.Logger(origin="main")
+    logger = utils.Logger(origin="main", print_to_console=True)
```

Comment:
I guess this is only needed for the simulated device. I would add a if clause here.

Reply:
yes, good catch!

Reply:
fixed it now, only logs to console if in simulate mode